root/static/scripts/release-group/components/SetCoverArtForm.js

```diff
@@ -0,0 +1,209 @@
+      if (clickedElement.nodeName === 'BDI') {
+        return;
+      }
+      // Avoid selecting by clicking on the image / opening lightbox
+      if (clickedElement.nodeName === 'IMG') {
+        return;
+      }
+    }
+    event.preventDefault();
+    const isSelectable = event.currentTarget.dataset.selectable === 'true';
+    if (!isSelectable) {
+      return;
+    }
+    const gid = event.currentTarget.dataset.gid;
+    setSelected(gid);
+  };
+
+  return (
+    <form className="set-cover-art" id="set-cover-art" method="post">
+      <HiddenField field={form.field.release} value={selected} />
+
+      <p>
+        {exp.l(
+          `Only releases with a front cover on the {caa|Cover Art Archive}
+            can be selected.`,
+          {caa: '//coverartarchive.org'},
+        )}
+      </p>
+      <div className="row" id="set-cover-art-position-row">
+        <FieldErrors field={form.field.release} />
+        <div id="set-cover-art-images">
+          {allReleases.map((release, index) => {
+            const image = artwork[release.id];
+            const releaseCatnos =
+              <ReleaseCatnoList labels={release.labels} />;
+            const releaseCountries =
+              <ReleaseCountryList events={release.events} />;
+            const releaseDates =
+              <ReleaseDateList events={release.events} />;
+            const releaseLabels =
+              <ReleaseLabelList labels={release.labels} />;
+            return (
+              <div
+                className={
+                  'editimage' + (release.gid === selected ? ' selected' : '')
+                }
+                data-gid={release.gid}
+                data-selectable={image ? 'true' : 'false'}
+                key={index}
+                onClick={selectCover}
+              >
+                <div className="cover-image">
+                  {image ? (
+                    <Artwork artwork={image} />
+                  ) : (
+                    <img
+                      src="/static/images/no-cover-art.png"
+                      title={l('No cover art available.')}
+                    />
+                  )}
+                </div>
+                <div className="release-description">
+                  <p>
+                    <EntityLink entity={release} />
+                    <br />
+                    {addColonText(l('Artist'))}
+                    {' '}
+                    <ArtistCreditLink artistCredit={release.artistCredit} />
+                  </p>
+                  {releaseCountries || releaseDates ? (
+                    <p>
+                      {releaseDates ? (
+                        <>
+                          {addColonText(l('Date'))}
+                          {' '}
+                          {releaseDates}
+                          <br />
+                        </>
+                      ) : null}
+                      {releaseCountries ? (
```

Comment:
This is testing the React element you created, which will always be truthy. You'll need to check `release.events` instead (same for other places here where you check a React element).